db/migrate/20180823081502_remove_foreman_docker_support.rb

```diff
@@ -0,0 +1,27 @@
+class RemoveForemanDockerSupport < ActiveRecord::Migration[5.1]
+  def change
+    if table_exists?(:docker_images)
+      remove_column :docker_images, :katello_uuid
+      remove_column :docker_images, :katello_repository_id
+    end
+    if table_exists?(:docker_tags)
+      remove_index :docker_tags, :name => :katello_docker_tag_repository_id
+      remove_column :docker_tags, :katello_repository_id
+    end
+    if table_exists?(:docker_container_wizard_states_images)
+      remove_column :docker_container_wizard_states_images, :capsule_id
+    end
+    if table_exists?(:containers)
+      remove_column :containers, :capsule_id
+    end
+    if table_exists?(:containers)
+      drop_table :katello_repository_docker_images
+    end
+    if foreign_key_exists?(:katello_repository_docker_images, :name => "docker_images")
+      remove_foreign_key :katello_repository_docker_images, :name => "docker_images"
+    end
+    if foreign_key_exists?(:katello_repository_docker_images, :name => "katello_repositories")
+      remove_foreign_key :katello_repository_docker_images, :name => "katello_repositories"
+    end
```

Comment:
I don't think there's any way these can exist, because those tables were deleted in ~2016?